setup.py

```diff
@@ -47,9 +47,6 @@
 ]
 
 extras_require = {
-    'cors': [
```

Comment:
Hm, on second thought, this will probably break quite some code because I think we have this dependency many places. Perhaps leaving the 

```
'cors': []
```

To prevent breaking changes?